Add BBE for `int:range` (#3324)
ballerina-platform/ballerina-distribution: examples/int-range/int_range.md

# int:range function

> `int:range` Langlib function returns an iterable which iterates over a range of integers. This is particularly useful when creating a loop with a decrementing looping variable or decrement/increment with a step.  

Reviewer: ```suggestion
The `int:range` lang library function returns an iterable object which iterates over a range of integers. This is particularly useful when creating a loop with a decrementing looping variable or decrement/increment with a step.  
```

> The `int:range` lang library function returns an iterable object which iterates over a range of integers. This is particularly useful when creating a loop with a decrementing looping variable or decrement/increment with a step.  

Reviewer: ```suggestion
The `int:range` lang library function returns an iterable object which iterates over a range of integers. This is particularly useful when creating a loop with a decrementing looping variable or increment/decrement with a step.  
```

> The `int:range` lang library function returns an iterable object which iterates over a range of integers. This is particularly useful when creating a loop with a decrementing looping variable or increment/decrement with a step.  

Reviewer 2: ```suggestion
The `int:range` lang library function returns an iterable object, which iterates over a range of integers. This is particularly useful when creating a loop with a decrementing looping variable or increment/decrement with a step.  
```
